app/components/new-signup/select.hbs

```diff
@@ -0,0 +1,32 @@
+<div class='user-details'>
+  <h2 class='user-details__heading' data-test-signup-form-label>
+    {{this.label}}
+  </h2>
+  <div class='user-details__field role-details__field'>
+    <select
+      name='role'
+      id='role'
+      {{on 'change' this.selectFieldChanged}}
+      data-test-signup-form-select
+    >
+      <option disabled selected>Choose role</option>
+      <option value='developer' selected={{this.developer}}>Developer</option>
+      <option value='designer' selected={{this.designer}}>Designer</option>
+      <option value='mavens' selected={{this.mavens}}>Mavens</option>
+      <option value='productManager' selected={{this.productManager}}>
+        Product Manager
+      </option>

```

Comment:
We should convert the value and name required for options into an object and store it in a constant file and use it here
this should be converted to {{each}} to render the options (loop over the options object)

Reply:
As requirement change now...can you please check my latest commit